### ISSUE TITLE
does not compile, `cargo update` fixes it tho

### ISSUE THREAD

**Issue by quantenzitrone** [svartalf/rust-battop#29]
When compiling, I get a really long error message that seems to be in some dependency: [error.txt](https://github.com/svartalf/rust-battop/files/13478633/error.txt)

Running `cargo update` seems to also update the faulty dependency.
I still get some warnings, but nothing fatal.

**Comment by dividebysandwich** [comment]
Yep, there's an error in uom 0.23.1, and cargo update bumps that up to 0.30.0 which fixes it.

**Comment by loodvn** [comment]
I think it would be fixed once https://github.com/svartalf/rust-battop/pull/22 is merged
